apis/kueue/v1alpha1/tas_types.go

```diff
@@ -27,3 +53,57 @@ const (
 	// The label is set when starting the Job, and removed on stopping the Job.
 	PodSetLabel = "kueue.x-k8s.io/podset"
 )
+
+// TopologySpec defines the desired state of Topology
+type TopologySpec struct {
+	// TopologyLevels define the levels of topology.
```

Comment:
```suggestion
	// levels define the levels of topology.
```

Let's use the YAML field name same as other APIs since this is API comment and these are referenced by users.

Reply:
Done